src/evolutionaryAlgorithm/EvolutionSimulator.py

```diff
@@ -52,11 +53,19 @@ def run(self):
     def initialize_population(self) -> List[IndividualInterface]:
         return self.initializator.initialize()
 
+    def markPopulation(self, population: List[IndividualInterface]) -> List[float]:
+        fitness = []
+        for el in population:
+            i = 0
+        #    i = ocena(el)
+            fitness.append(i)
+        return fitness
+
     def endSimulation(self) -> bool:
         return self.isEndSimulation
 
-    def selectParents(self, population: List[IndividualInterface]) -> List[IndividualInterface]:
-        return self.parentSelector.marry(population)
+    def selectParents(self, population: List[IndividualInterface], fitness: List[float]) -> List[IndividualInterface]:
+        return self.parentSelector.getParents(population, fitness)
```

Comment:
Dlaczego tu dalej jest fitness

Reply:
bo jeszcze nie ma zrobionej oceny, jak będzie ona zrobiona to szybko to zmienię